src/main/java/seedu/address/model/person/Person.java

```diff
@@ -26,19 +26,21 @@ public class Person {
     private final Set<Tag> tags = new HashSet<>();
     private final AdmissionDate admissionDate;
     private final Ward ward;
+    private final Remark remark;
 
     /**
      * Every field must be present and not null.
      */
     public Person(Name name, Set<Tag> tags,
-                  Dob dob, Ic ic, AdmissionDate admissionDate, Ward ward) {
-        requireAllNonNull(name, dob, ic, admissionDate, ward);
+                  Dob dob, Ic ic, AdmissionDate admissionDate, Ward ward, Remark remark) {
+        requireAllNonNull(name, dob, ic, admissionDate, ward, remark, tags);
```

Comment:
I believe remarks was optional, unless I am wrong. Should we exclude remarks from requireNonNull?